frontend/src/components/ide/AIAssistant/components/SettingsPanel.jsx

```
@@ -110,6 +124,119 @@ export function SettingsPanel({ settings, onSave }) {
+            >
+                {showAdvanced ? <ChevronUp size={16} /> : <ChevronDown size={16} />}
+                Расширенные настройки
+            </button>
+
+            {showAdvanced && (
+                <div className="space-y-3 pt-2 border-t">
+                    <div>
+                        <div className="flex justify-between items-center mb-1">
+                            <label className="text-sm font-medium">Температура</label>
+                            <span className="text-sm text-muted-foreground">{temperature}</span>
+                        </div>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)